lib/remote_helper.py

```diff
@@ -638,14 +640,15 @@ def copy_deployment_credentials(cls, host, options):
     # credentials, otherwise the AppScale VMs won't be able to interact with
     # GCE.
     if options.infrastructure and options.infrastructure == 'gce':
-      if os.path.exists(LocalState.get_client_secrets_location( \
-          options.keyname)):
-        cls.scp(host, options.keyname, LocalState.get_client_secrets_location(
-          options.keyname), '/etc/appscale/client_secrets.json',
-          options.verbose)
-      cls.scp(host, options.keyname, LocalState.get_oauth2_storage_location(
-        options.keyname), '/etc/appscale/oauth2.dat', options.verbose)
-
+      secrets_location = LocalState.get_client_secrets_location(options.keyname)
+      if not os.path.exists(secrets_location):
+        raise AppScaleException('{} does not exist.'.format(secrets_location))
+      secrets_type = GCEAgent.get_secrets_type(secrets_location)
+      cls.scp(host, options.keyname, secrets_location,
+        '/etc/appscale/client_secrets.json', options.verbose)
```

Comment:
Can we pull /etc/appscale from another module?
If it's not defined as a constant somewhere, maybe we can define it in lib/remote_helper?